content/courses/onchain-development/intro-to-anchor.md

```diff
@@ -10,24 +10,35 @@ description: "Create your first Solana onchain program in Anchor."
 
 ## Summary
 
-- **Programs** on Solana have **instruction handlers** that execute instruction
-  logic.
+- **Programs** on Solana have **instruction handlers**, which are functions that
+  take arguments from incoming instructions. They are the entry point for any
+  operation in a program.
 - **Rust** is the most common language for building Solana programs. The
   **Anchor** framework takes care of common grunt work - like reading data from
   incoming instructions, and checking the right accounts are provided - so you
   can focus on building your Solana program.
 
 ## Lesson
 
-Solana's ability to run arbitrary executable code is part of what makes it so
-powerful. Solana programs, similar to "smart contracts" in other blockchain
-environments, are quite literally the backbone of the Solana ecosystem. And the
-collection of programs grows daily as developers and creators dream up and
-deploy new programs.
+Before we begin, make sure you have Anchor installed. You can follow this lesson
+on [local-setup](./local-setup.md).
+
+Solana's capacity to execute arbitrary code is a key part of its power. Solana
+programs, a.k.a "smart contracts", are the very foundation of the Solana
```

Comment:
```suggestion
Solana's capacity to execute arbitrary code is a key part of its power. Solana
programs (sometimes called "smart contracts"), are the very foundation of the Solana
```

Reply:
We generally try to avoid smart contract and focus on programs, see CONTRIBUTING.